Jenkinsfile

```diff
@@ -20,7 +20,7 @@ def rocmnode(name) {
 
 
 
-def cmake_build(compiler, flags, prefixpath="/opt/rocm"){
+def cmake_build(compiler, flags, env4make, prefixpath){
```

Comment:
Perhaps: `makeenv` or `makeEnv`

Reply:
This stuff is intended to allow to modifying environment for make (e.g. `MIOPEN_LOG_LEVEL=5`) so "environment for make" -> `env4make`.  To me, `makeEnv` sound like "make environment", beginning with a noun.

Of course I can change to makeEnv, if you wish, just confirm please.

Reply:
Just a suggestion, since we never use variables in that format. I trust your judgement